Scripts/Runtime/Entities/TaskDriver/TaskSet/TaskData/DataStream/Cancellation/CancelProgressFlowNode.cs

```diff
@@ -19,13 +22,11 @@ internal class CancelProgressFlowNode : AbstractAnvilBase
         private readonly ITaskSetOwner m_TaskSetOwner;
         private readonly CancelProgressFlowNode m_Parent;
         private readonly ActiveLookupData<EntityProxyInstanceID> m_ProgressLookupData;
-        private readonly ActiveLookupData<EntityProxyInstanceID> m_ParentProgressLookupData;
         private readonly PendingData<EntityProxyInstanceWrapper<CancelComplete>> m_CancelCompleteData;
         private readonly DataTargetID m_CancelCompleteDataTargetID;
 
+        private OwnerSharedDataVersion m_LastProcessedDataVersion;
```

Comment:
Depends on: `OwnerSharedDataVersion` comment below.

```suggestion
        private readonly OwnerSharedDataVersion m_LastProcessedDataVersion;
```

Could make readonly?

Reply:
We COULD do this but there are two drawbacks that make this a bad idea
 - Would have to add `readonly` to the `ProgressLookupDataVersion` getter/setter. This is valid because the property doesn't actually mutate the struct but seems odd given that there's a setter present.
 - Calling `Dispose()` would cause the struct to be copied on each call. This would circumvent the whole purpose of the struct existing because the managed reference wouldn't be null out in the version stored on the field.

Reply:
Yeah it was depending on https://github.com/decline-cookies/anvil-unity-dots/pull/268/files#r1234045416 but I couldn't link to a pending comment.

Reply:
now a class and readonly.